### PR TITLE
Delete a Downloadable file for a Product

### LINKED ISSUE #2758: Add/Edit Products M5: Downloadable files Step 3 - delete downloadable file for a product
As part of the M5 release, we need to provide the option to delete a downloadable file for a product.

| Tap the bar button             |  Delete action |
:-------------------------:|:-------------------------:
iOS-add-price-2 | iOS-add-price

**Note that this issue only takes care of updating existing downloadable files for a product. The option to add new files will be taken care of in another issue and is not part of the scope of this issue.**

### PR DESCRIPTION
**Pull request by pmusolino**
Fixes #2758 

## Description
As part of Products M5 we would allow users to delete a downloadable file from the downloadable file screen detail.
In this PR, I implemented the action sheet that shows the `Delete` button + the action.

## Testing
1. Open a product with downloadable files.
2. Open a downloadable file detail.
3. Tap the "more" button item on the navbar, that will be visible only when you are editing a downloadable file.
4. Press the `Delete` button.
5. Update the product, and make sure that the changes will be applied correctly, removing the specified downloadable file.

## GIF

Update release notes:
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Review by Garance91540 — APPROVED** [review]
👏

**Comment by loremattei** [comment]
Hey there! I'm moving this to 5.4 as 5.3 has been cut. If you still want this to make it to 5.3, please feel free to ping me.

**Comment by pmusolino** [comment]
CI is failing because of [this](https://a8c.slack.com/archives/C6H8C3G23/p1603115546475700).

File: WooCommerce/Classes/ViewRelated/Products/Edit Product/Downloadable Files/Add or Edit File/ProductDownloadFileViewController.swift [hunk 1]
```diff
@@ -53,6 +60,28 @@ final class ProductDownloadFileViewController: UIViewController {
 //
 extension ProductDownloadFileViewController {
 
+    @objc private func presentMoreActionSheetMenu(_ sender: UIBarButtonItem) {
+        let menuAlert = UIAlertController(title: nil, message: nil, preferredStyle: .actionSheet)
+        menuAlert.view.tintColor = .text
+
+        let deleteTitle = NSLocalizedString("Delete",
```
Comment (Ecarrion) [inline WooCommerce/Classes/ViewRelated/Products/Edit Product/Downloadable Files/Add or Edit File/ProductDownloadFileViewController.swift]: Should we move this to a localization enum?

**Review by Ecarrion — APPROVED** [review]
Looks good!

File: WooCommerce/Classes/ViewRelated/Products/Edit Product/Downloadable Files/Add or Edit File/ProductDownloadFileViewController.swift (hunk 1 above)
Comment (pmusolino) [reply WooCommerce/Classes/ViewRelated/Products/Edit Product/Downloadable Files/Add or Edit File/ProductDownloadFileViewController.swift]: Yes, this was already done in this other PR https://github.com/woocommerce/woocommerce-ios/pull/3014 👍

**Pull request merged**